src/main/java/me/gnat008/perworldinventory/data/metadata/PWIMetaDataValueBasic.java

```diff
@@ -0,0 +1,33 @@
+package me.gnat008.perworldinventory.data.metadata;
+import me.gnat008.perworldinventory.PerWorldInventory;
+
+import javax.inject.Inject;
+
+public class PWIMetaDataValueBasic<T> extends PWIMetaDataValueAbstract<T> {
```

Comment:
I left it there as a way to easily add primitive/simple data to players, for future uses. (In the beginning I was gonna use it for my data, but then I realized I actually could invalidate my data, and went with implementing it in another way)

Should this be removed entirely, or a way to add primitive data to players, using the `PWIMetaDataManager`, which I'm about to rewrite`?